portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,100 @@
 <!DOCTYPE html>
 <html>
+  <body style="background-color:#ccebc3;"></body>
```

Comment:
YESSSS, so much prettier!!! Thanks for getting this change in! You still have a few `style=` left in this file, I would just ctrl+f for them and try to replace them all